src/openforms/config/constants.py

```diff
@@ -53,3 +53,28 @@ class CSPDirective(DjangoChoices):
     # CSP_UPGRADE_INSECURE_REQUESTS  # Include upgrade-insecure-requests directive. A boolean. False See: upgrade-insecure-requests
     # CSP_BLOCK_ALL_MIXED_CONTENT  # Include block-all-mixed-content directive. A boolean. False See: block-all-mixed-content
     # CSP_INCLUDE_NONCE_IN  # Include dynamically generated nonce in all listed directives, e.g. CSP_INCLUDE_NONCE_IN=['script-src'] will add 'nonce-<b64-value>' to the script-src directive.
+
+
+class UploadFileType(DjangoChoices):
+    ALL = ChoiceItem("*", label="All")
```

Comment:
Please convert all vars to lowercase (ALL -> all) as a convention.

For the `label="All"` this can be translated as well: `_("All")`